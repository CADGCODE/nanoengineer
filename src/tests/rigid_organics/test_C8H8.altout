======= test_C8H8.test =======
TYPE struct
======= exitvalue =======
0
======= structurematch =======
1
======= stderr =======
cutover to fine tolerance at 7
== structure comparison ==
error reading atom 0 from test_C8H8.xyzcmp
could not read base positions file from -B<filename>
======= stdout =======
iters per frame = 10
number of frames = 100
timestep = 1.000000e-16
temp = 0.000000
dump as text
< test_C8H8.mmp  > test_C8H8.xyz
final RMS gradient=15407.984824 after 10 iterations
== structure comparison ==
======= test_C8H8.trc =======
# nanoENGINEER-1.com Simulator Trace File, Version 050310
#
# Input File:test_C8H8.mmp
# Output File: test_C8H8.xyz
# Trace File: test_C8H8.trc
# Number of Atoms: 16
# Number of Frames: 100
# Steps per Frame: 10
# Temperature: 300.0
# 
# 0 columns:
#
0 4251.36 5876.70
1 29096.24 39410.77
2 12559.83 17699.86
3 16553.38 22499.57
4 15052.92 20739.21
5 15531.70 21308.44
6 15362.80 21109.05
7 15421.24 21178.46
8 15400.97 21154.38
9 15407.98 21162.71
10 15407.98 21162.71
# Done: Minimization final rms: 15407.984824, highForce: 21162.714841
======= test_C8H8.xyz =======
16
RMS=15407.984824
C -0.603999 0.400390 1.415139
C -1.107162 1.121846 -0.164640
C 0.635572 1.280111 -0.700677
C 0.599130 -0.369365 -1.441536
C -0.631880 -1.288879 0.689386
C 1.174772 0.560050 0.924517
C -1.176047 -0.561704 -0.934272
C 1.108994 -1.123426 0.203645
H -0.952036 0.638416 2.294537
H -1.805691 1.805735 -0.303273
H 1.039134 2.078902 -1.105224
H 0.957565 -0.641348 -2.313141
H -1.037650 -2.079870 1.096408
H 1.891325 0.910935 1.488902
H -1.897707 -0.914929 -1.498497
H 1.809679 -1.807864 0.286726
