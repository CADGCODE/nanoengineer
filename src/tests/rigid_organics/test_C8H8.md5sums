stdout 5EB17426392307C80DA1D19E2AB0FCC3
stderr D41D8CD98F00B204E9800998ECF8427E
test_C8H8.trc 1781D370846019977E77804E4D068899
test_C8H8.xyz A6B1148DCC05F55B9D9F0BF52586EACA
