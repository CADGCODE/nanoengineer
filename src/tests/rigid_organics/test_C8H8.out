======= test_C8H8.test =======
TYPE struct
======= exitvalue =======
0
======= structurematch =======
0
======= stderr =======
cutover to fine tolerance at 7
== structure comparison ==
